near-accounts/src/accounts.rs

```diff
@@ -499,18 +497,21 @@ pub async fn get_account_balance(
     let protocol_config = provider
         .experimental_protocol_config(block_reference)
         .await?;
-    let cost_per_byte = BigInt::from(protocol_config.runtime_config.storage_amount_per_byte);
+    // let cost_per_byte = BigInt::from(protocol_config.runtime_config.storage_amount_per_byte);
 
     let state = state(provider, account_id).await?;
 
-    // Assuming state.storage_usage, state.locked, and state.amount are already BigInt or can be converted to BigInt
-    let state_staked = BigInt::from(state.storage_usage) * &cost_per_byte;
-    let staked = BigInt::from(state.locked);
-    let total_balance = BigInt::from(state.amount) + &staked;
-    let available_balance = if staked > state_staked {
-        &total_balance - &staked
+    let staked = state.locked;
+    let state_staked = protocol_config
+        .runtime_config
+        .storage_amount_per_byte
+        .mul(state.storage_usage as u128);
+    let total_balance = staked.add(state.amount);
+
+    let available_balance = if staked.ge(&state_staked) {
+        total_balance.sub(staked)
     } else {
-        &total_balance - &state_staked
+        total_balance.sub(state_staked)
     };
 
```

Comment:
BigInt was used here because the value of staked and other balances here are in YoctoNear so atleast 24 zeroes. 

- Are you sure without BigInt we can handle the arithmetic operations here?
- Did you run tests if this function works?

Reply:
So all the values used in this calculation are of type u128 which has a range that is quite higher than the needed, please check the max value https://doc.rust-lang.org/std/primitive.u128.html#associatedconstant.MAX .

Yes, I have run the example and it's working perfectly fine.